hack/make-rules/test-integration.sh

```diff
@@ -94,7 +97,7 @@ checkEtcdOnPath
 trap cleanup EXIT
 
 # Convert the CSV to an array of API versions to test
-IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
+IFS=';' read -ar apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
Why did it change and what did it fix:
```
In ./hack/make-rules/test-integration.sh line 97:
IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
        ^--^ SC2162: read without -r will mangle backslashes.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2162
```

Reply:
`read -ar` cannot run on my environment. I think `read -ra` is correct.

Reply:
Good catch :+1: 
I think you are right!